### PR TITLE
Add additional ignoring for latest hadolint version

### PR DESCRIPTION
**Pull request by jacob-delgado**
Needed for master-2021-02-22T14-24-32

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG VM_IMAGE_VERSION=8
 FROM centos:${VM_IMAGE_VERSION}
 
-# hadolint ignore=DL3005,DL3008
+# hadolint ignore=DL3005,DL3008,DL3031,DL3032,DL3033
```
Comment (ericvn) [inline pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos:4]: Should we actually fix the 3031 and 3032 errors?

Comment (jacob-delgado) [reply pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos:4]: This is only used for the echo app that is only used in integration testing. This isn't for end users.  I can fix it if you'd like.

For DL3032 `yum clean all` is being run, but the linter doesn't pick up.

**Review by howardjohn — APPROVED** [review]

File: pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos (hunk 1 above)
Comment (ericvn) [reply pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos:4]: I don't think the `yum clean all` is being run (missing &&).

Comment (ericvn) [reply pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos:4]: Fixed that issue in https://github.com/istio/istio/pull/30998

**Pull request merged**